x/stakeibc/genesis.go

```diff
@@ -10,48 +10,23 @@ import (
 // InitGenesis initializes the capability module's state from a provided genesis
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
-	// TODO: TEST-10
-	// Set if defined
-	if genState.IcaAccount != nil {
-		k.SetICAAccount(ctx, *genState.IcaAccount)
+	for _, hostZone := range genState.HostZoneList {
+		k.SetHostZone(ctx, hostZone)
 	}
-	// Set all the hostZone
-	for _, elem := range genState.HostZoneList {
-		k.SetHostZone(ctx, elem)
+	for _, epochTracker := range genState.EpochTrackerList {
+		k.SetEpochTracker(ctx, epochTracker)
 	}
 
-	// Set hostZone count
-	k.SetHostZoneCount(ctx, genState.HostZoneCount)
```

Comment:
If we're removing HostZoneCount here, I think we should also remove it from the protos. I don't think HostZoneCount is used anywhere in the repo, so this should be safe! 

Also reasonable if we want to split this into another PR given it's not directly related to this issue, but feels like a small change that could be bundled in

Reply:
I agree! Removing it now